src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/types/concept.types.js

```diff
@@ -1,4 +1,6 @@
 // @flow
+import { typeof dataElementTypes } from '../../../../../metaData';
+
 type TeiRecord = {| [id: string]: any |};
 
 export type TeiRecords = {| [teiId: string]: TeiRecord |};
```

Comment:
out of scope but looks like this is unused?

Reply:
Yes, got a bit ahead of myself there, it is used in the next PR so I hope it's ok if I leave it there.